packages/mdc-toolbar/README.md

```diff
@@ -20,13 +19,32 @@ Wrap the items with `mdc-toolbar` class in following way:
 
 ```html
 <header class="mdc-toolbar">
-  <section class="mdc-toolbar__section mdc-toolbar__section--align-start">
-    <a class="material-icons">menu</a>
-    <span class="mdc-toolbar__title">Title</span>
-  </section>
+  <div class="mdc-toolbar__row">
+    <section class="mdc-toolbar__section mdc-toolbar__section--align-start">
+      <a class="material-icons">menu</a>
+      <span class="mdc-toolbar__title">Title</span>
+    </section>
+  </div>
 </header>
 ```
 
+You can wrap second row such as mdc-tabs within mdc-toolbar:
+
+```html
+<header class="mdc-toolbar">
+  <div class="mdc-toolbar__row">
+    <section class="mdc-toolbar__section mdc-toolbar__section--align-start">
+      <a class="material-icons">menu</a>
+      <span class="mdc-toolbar__title">Title</span>
+    </section>
+  </div>
+  <div class="mdc-toolbar__row">
+    <nav class="mdc-tabs">...</nav>
```

Comment:
Hopefully! But I'd rather not include this in here until it's landed, given that it may confuse users coming to this documentation for the first time when they don't immediately see `mdc-tabs` in our codebase.